plugin/al/mayautils/AL/maya/event/MayaEventManager.cpp

```diff
@@ -921,27 +926,31 @@ void MayaEventHandler::initDagMessage(MayaCallbackInfo& cbi)
         cbi.mayaCallback
             = MDagMessage::addAllDagChangesCallback(bindMessageParentChildFunction, &cbi);
         break;
-    case DagMessage::kAllDagChangesDagPath: /* cbi.mayaCallback =
-                                               MDagMessage::addAllDagChangesDagPathCallback(bindNodeStringBoolFunc,
-                                               &cbi); */
+    case DagMessage::
+        kAllDagChangesDagPath: /* cbi.mayaCallback =
+                                  MDagMessage::addAllDagChangesDagPathCallback(bindNodeStringBoolFunc,
+                                  &cbi); */
         break;
     case DagMessage::kInstanceAdded:
         cbi.mayaCallback = MDagMessage::addInstanceAddedCallback(bindParentChildFunction, &cbi);
         break;
-    case DagMessage::kInstanceAddedDagPath: /* cbi.mayaCallback =
-                                               MDagMessage::addInstanceAddedDagPathCallback(bindNodeStringBoolFunc,
-                                               &cbi); */
+    case DagMessage::
+        kInstanceAddedDagPath: /* cbi.mayaCallback =
+                                  MDagMessage::addInstanceAddedDagPathCallback(bindNodeStringBoolFunc,
+                                  &cbi); */
         break;
     case DagMessage::kInstanceRemoved:
         cbi.mayaCallback = MDagMessage::addInstanceRemovedCallback(bindParentChildFunction, &cbi);
         break;
-    case DagMessage::kInstanceRemovedDagPath: /* cbi.mayaCallback =
-                                                 MDagMessage::addInstanceRemovedDagPathCallback(bindNodeStringBoolFunc,
-                                                 &cbi); */
+    case DagMessage::
+        kInstanceRemovedDagPath: /* cbi.mayaCallback =
```

Comment:
This is pretty ugly...wasn't expecting clang-format to break it this way. Must have something to do with commented code after.

Something to remember and revisit eventually.

Reply:
Why not put it back to what it was before and use the clang-format off/on comment?

Reply:
I don't want developers to have to fallback to disabling clang-format in too many cases. We will update clang-format version at one point. This will be a good opportunity to revising some of the clang-format settings and prevent this type of formatting.

For that reason, I'm not going to use the clang-format off/on comment.